CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.0.0-rc.18 (2017-09-03)
+
+- Created the `--no-build` flag for `skyux e2e`, which will skip the build step before running end-to-end tests. [#262](https://github.com/blackbaud/skyux-builder/pull/262)
```

Comment:
I'd change ", which will skip" to "`to skip" (including deleting the comma). ... That said, it's just a wording preference, so no big deal if you prefer to leave it as is.